GenesToPhenotypes.ipynb

```diff
@@ -107,14 +114,155 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### For each in gene in the list fetch the screens, plates, images\n",
-    "and then phenotypes associated with the images\n",
-    "The results are both printed to the screen and saved in a .csv file"
+    "### Find images for each gene specified\n",
+    "For each gene, search of images either in plates or datasets then search for phenotypes associated with the images.\n",
+    "The results are both printed to the screen and saved in a CSV file."
```

Comment:
```suggestion
    "The results are saved in a CSV file."
```

Suggesting that because the printing is not happening in this cell, but much lower now (which is fine, just need to adjust the comment(